server/tests/api.test.js

```diff
@@ -4,37 +4,49 @@ import Database from "../database.js"
 
 const databaseDisabledError = { error: 'Database not implemented' };
 const incorrectQueryParamsError = { error: 'Incorrect query parameters' };
+console.assert(process.env.NODE_ENV == "test", "This test suite should only be run in the test environment");
 
 beforeAll(async () => {
-    process.env.DATABASE_ENABLED = true;
+    // process.env.DATABASE_ENABLED = true;
     await API.init();
+    await Database.init();
+
+    // would be good to do before each test
+    // but it's so slow and none of the tests are mutating anyway
+    // it would be good to make this not random, but Math.random has no seed function
+    await Database.loadTestData();
```

Comment:
before mutations, it would be good to get import/export working with a proper test database, because the building is super slow which is why I don't run it before each test. It was taking about 10 seconds on my computer
neo4j.com/labs/apoc/4.3/export/json/ might work but i haven't really looked into it yet. it has instructions for both importing and exporting